sys/reaper/reaper_utils_linux.go

```diff
@@ -14,30 +14,17 @@
    limitations under the License.
 */
 
-package sys
+package reaper
 
 import (
 	"unsafe"
 
 	"golang.org/x/sys/unix"
 )
 
-// If arg2 is nonzero, set the "child subreaper" attribute of the
-// calling process; if arg2 is zero, unset the attribute.  When a
-// process is marked as a child subreaper, all of the children
-// that it creates, and their descendants, will be marked as
-// having a subreaper.  In effect, a subreaper fulfills the role
-// of init(1) for its descendant processes.  Upon termination of
-// a process that is orphaned (i.e., its immediate parent has
-// already terminated) and marked as having a subreaper, the
-// nearest still living ancestor subreaper will receive a SIGCHLD
-// signal and be able to wait(2) on the process to discover its
-// termination status.
-const setChildSubreaper = 36
-
 // SetSubreaper sets the value i as the subreaper setting for the calling process
 func SetSubreaper(i int) error {
-	return unix.Prctl(setChildSubreaper, uintptr(i), 0, 0, 0)
+	return unix.Prctl(unix.PR_SET_CHILD_SUBREAPER, uintptr(i), 0, 0, 0)
 }
 
 // GetSubreaper returns the subreaper setting for the calling process
```

Comment:
I don't actually see this function being used anywhere; it was added as part of https://github.com/containerd/containerd/commit/da1b5470cd26e9e2b4b1cd0100c876ce8db87471, but wasn't used there as well (perhaps it was originally meant as a generic utility?).

Reply:
Looks like it was copied from libcontainer, but not in use in this repository https://github.com/opencontainers/runc/blob/12ba2a7320545ff6cf2bcc87f8bae069a8fda3db/libcontainer/system/linux.go#L141-L150

Reply:
just leave it.  other external things can use it